### PR TITLE
Fix permanent displaying

### PR DESCRIPTION
**Pull request by nickvergessen**
Regression in conflict resolution from #106

See #107  and #111 for discussion.

### CODE REVIEW

**Review by skjnldsv — COMMENTED** [review]
I don't get it, Toastify requires -1 for permanent
https://github.com/apvarun/toastify-js/blob/master/README.md#api

**Comment by nickvergessen** [comment]
https://github.com/apvarun/toastify-js/commit/4e675736f1154d42c49c133d9f85a2ae49db2ad3

okay this changed with 1.8.0 then let me update the readme instead.

**Review by skjnldsv — APPROVED** [review]
sure

File: lib/toast.ts [hunk 1]
```diff
@@ -175,7 +180,7 @@ export function showUndo(text: string, onUndo: Function, options?: ToastOptions)
 
 	options = Object.assign(options || {}, {
 		// force 10 seconds of timeout
-		timeout: 1000,
+		timeout: 10000,
```
Comment (nickvergessen) [inline lib/toast.ts:183]: Because of this I would actually do a 2.0.1 release.

Comment (skjnldsv) [reply lib/toast.ts:183]: Right!

**Comment by skjnldsv** [comment]
So, this actually broke Undo, because 10000 timeout is now multiplied by 1000.
Why are we using seconds instead of milliseconds?

Also docs still says to use 0.
https://nextcloud.github.io/nextcloud-dialogs/interfaces/_toast_.toastoptions.html#timeout...

@nickvergessen @ChristophWurst

**Comment by skjnldsv** [comment]
Fixed by https://github.com/nextcloud/nextcloud-dialogs/pull/210

**Pull request merged**